### PR TITLE
Cssshapes

### PR DESCRIPTION
**Pull request by devnook**
Adding a CSS Shapes level 1 sample.

### CODE REVIEW

**Comment by jeffposnick** [comment]
This is what I'm seeing for the last two examples (using `content-box` and `padding-box`) using Chrome 38. Just wanted to make sure it's intended.

image

File: css-shapes/index.html [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+        shape-outside: polygon(0 0, 100% 0, 100% 100%, 0 100%) content-box;
+      }
+    </style>
+    <div class="shape box content-box">
+      <p>content</p>
+    </div>
+    <p>
+      This shape uses <code>content-box</code>, defined in the
+      <code>shape-outside</code> property. The content is in red,
+      content in yellow and margin area in light blue.
```
Comment (jeffposnick) [inline css-shapes/index.html]: Should this say "_padding in yellow_" instead of "_content in yellow_"?

Comment (devnook) [inline css-shapes/index.html]: Fixed. as for the flowing text, it is there to show different spacing of the element, so it's intended.

**Comment by devnook** [comment]
PTAL

**Pull request merged**